meson.build

```diff
@@ -288,6 +250,49 @@ foreach feature_name, data : features
 
 endforeach
 
+# Define warning flags for c and cpp
```

Comment:
Why do you need to move this block?

Reply:
pytorch and caffe2 do not comply with `-wredundant-declarations`. The patch tries to add it when pytorch and caffe2 extensions aren't compiled. Could not find a way to remove already added project arguments.
Move back the default declared flags to the original location.